src/main/webapp/sass/_core.scss

```diff
@@ -2501,6 +2501,9 @@ i.fa.small-icon {
   .min-width-200-unless-xs {
     min-width: 200px;
   }
+  .max-width-100-unless-xs {
```

Comment:
Could you rename to .max-width-100percent-unless-xs or rename min-width-200-unless-xs to min-width-200px-unless-xs?